pyproject.toml

```diff
@@ -1,24 +1,26 @@
-# https://pycqa.github.io/isort/
-[tool.isort]
-profile = "black"
-multi_line_output = 3
-line_length = 120
-skip_gitignore = true
+[tool.bandit]
```

Comment:
I don't think it's a good idea to let the linters remove the comments.

Reply:
I agree: the `pretty-format-toml` hook removes comments unless they’re at the top of the file.

Looking at [the code](https://github.com/macisamuele/language-formatters-pre-commit-hooks/blob/16762a19ddaa3a966ca7c4edd9ade6189f0df627/language_formatters_pre_commit_hooks/pretty_format_toml.py#L32) it uses the [toml_sort](https://github.com/pappasam/toml-sort) package which has an open issue https://github.com/pappasam/toml-sort/issues/11.

Reply:
How about removing `pretty-format-toml` until the issue is resolved and use a different hook?

Reply:
Considering that that hook does nothing but sorting the entries, we could add the comments back in and comment out the hook 👍🏼

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/91/commits/f8d4e61eed5c8cafa7ac3cd31d94c851ac13d064.